dispatch/dispatch.go

```diff
@@ -206,6 +206,8 @@ type AlertGroup struct {
 	Alerts   types.AlertSlice
 	Labels   model.LabelSet
 	Receiver string
+	GroupKey string
+	RouteID  string
```

Comment:
Needed to add `RouteID` here because we added it to the marker. Without it, we cannot see if a group is muted or not.